consensus/src/engine/mempool_config.rs

```diff
@@ -345,10 +336,11 @@ impl MempoolConfigBuilder {
 }
 
 // Note: never derive Default for Genesis data
-#[derive(Clone, Debug, Eq, PartialEq)]
-struct GenesisData {
-    round: u32,
-    time_millis: u64,
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+pub struct GenesisData {
+    /// will be aligned to become genesis round
+    pub start_round: u32,
+    pub time_millis: u64,
 }
```

Comment:
Would like to move this struct into `everscale-types` to become part of `ConsensusInfo` stored in `ShardStateUnsplit`, not in `BlockchainConfig`.
This will unify naming and usage: aligned genesis round is used only inside `tycho_consensus`, while `GlobalConfig` and ``ConsensusInfo` store unaligned `start_round`, that corresponds to `processed_up_to` from mc block.